src/modules/auth/credentials/credentials.scss

```diff
@@ -168,3 +181,98 @@
 		min-height: 1px;
 	}
 }
+
+// BULK ASSIGNMENT
+.bulk-actions-wraper {
+    display: grid;
+    grid-template-areas:
+        "a a c c"
+        "a a c c"
+		"b b d d"
+		"b b d d";
+
+    grid-template-rows: 1fr 1fr 1fr 1fr;
+	grid-template-columns: 1fr 1fr 1fr 1fr;
+    column-gap: 0.25rem;
+    row-gap: 0.25rem;
+	& .card-body {
+		height: 35vh !important;
+		overflow-y: auto;
+	}
+}
+
+.credentials-suspended {
+	color: #73818f!important;
```

Comment:
@petrKavulok why??? we have `$secondary` color for that. also you have wrong indentation here